HACKING

```diff
@@ -787,6 +787,9 @@ Building Plugins
 The geany-plugins autotools script automatically detects the
 installed system Geany and builds the plugins against that.
 
+Plugins will build with the version of GTK2/3 that the detected 
+Geany was built with.
```

Comment:
that's not clear (IMO): plugin will have to follow the *major* version, which is either *GTK2* or *GTK3*.  Here, I feel like your sentence suggests it'll build with whichever specific version of GTK2 *or* GTK3 Geany was built against (e.g. that if Geany was built against 2.24.10, plugins will use that exact version, same in case of 3.4.2, etc) instead of suggesting that plugins will use GTK2 if Geany does, and same with GTK3.

Reply:
hmm, ok, if it confuses someone who knows whats going on then what about:

"Plugins will build with either GTK2 or GTK3, whichever the detected Geany was built with."

Reply:
Better, sounds good to me.